meta-resin-common/recipes-connectivity/redsocks/files/0003-Add-OS-dependent-default-configuration-values.patch

```diff
@@ -0,0 +1,98 @@
+From a05247037a5d7be34d6df83411df08fe0f176210 Mon Sep 17 00:00:00 2001
+From: Andrei Gherzan <andrei@resin.io>
+Date: Fri, 20 Apr 2018 13:06:14 +0100
+Subject: [PATCH] Add OS dependent default configuration values
+
```

Comment:
@ZubairLK why is that? This configuration needs to match the ip tables configuration. Otherwise the redirection won't happen correctly.

Reply:
just that these should be in read from a config file rather than hard-coded.
But looks like redsocks is on v0.5 and maybe not as actively maintained as I assumed.
Code edits works for now